.github/workflows/examples.yml

```diff
@@ -54,6 +54,8 @@ jobs:
           IGNORES='chainermn_.*|fastai*|rapids_.*|botorch_.*'
         elif [ ${{ matrix.python-version }} = 3.8 ]; then
           IGNORES='chainermn_.*|dask_ml_.*|keras_.*|tensorboard_.*|tensorflow_.*|tfkeras_.*|fastai*|allennlp|rapids_.*'
+        elif [ ${{ matrix.python-version }} = 3.9 ]; then
+          IGNORES='allennlp_*|botorch_*|catalyst_*|catboost_*|chainer_*|chainermn_.*|dask_ml_.*|fastai*|gluon_*|haiku_*|keras_*|lightgbm_*|mxnet_*|optuna_search_cv_*|pytorch_ignite_*|pytorch_lightning_*|rapids_*|sb3_*|skimage_lbp_*|sklearn_*|skorch_*|tensorboard_*|tensorflow_*|tfkeras_*|xgboost_*'
```

Comment:
The target examples will be as follows, but many of them did not work as expected due to the lack of dependencies. Could you exclude them?
- examples/pruning/simple.py: did not work due to the lack of `scikit-learn`
- examples/enqueue_trial.py: did not work due to the lack of `scikit-learn`
- examples/quadratic_simple.py: OK 
- examples/visualization/plot_pareto_front.py: did not work due to the lack of `plotly`
- examples/visualization/plot_study.py: did not work due to the lack of `plotly`
- examples/pytorch_simple.py: OK
- examples/multi_objective/pytorch_simple.py: did not work due to the lack of `thop`

Reply:
lgtm other than this comment.